### PR TITLE
Add Guix and rde installation instructions

### PR DESCRIPTION
**Pull request by abcdw**
Thank you for making this nice notification center! We have SwayNotificationCenter packaged in [Guix](https://git.savannah.gnu.org/cgit/guix.git/tree/gnu/packages/wm.scm?h=a831efb52f49a8424a915f30486730e0fd4ba4e2#n1820) and [rde](https://git.sr.ht/~abcdw/rde/tree/3b81be46d3d6891663f857172bc00b46f0e1eafa/src/rde/features/wm.scm#L906). Added respective installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,23 @@ Bookworm and later:
 sudo apt install sway-notification-center
 ```
 
+### Guix
```
Comment (ErikReider) [inline README.md:154]: Don't forget to add links to the Table Of Contents

**Review by ErikReider — CHANGES_REQUESTED** [review]
Could you please also explain what rde is? I get that Guix is a package manager but not what rde has to do with it

**Comment by abcdw** [comment]
rde is a power user and developer friendly GNU/Linux distribution on top of Guix API. It provides a high-level declarative API, pre-configured batteries and quite deep integration between different components. Very roughly it's similiar to doom emacs, but reproducible and for the whole system.

**Review by ErikReider — APPROVED** [review]
Thanks! :)

**Pull request merged**